app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :categories, foreign_key: 'author_id', class_name: 'Category'
+  has_many :payments, foreign_key: 'author_id', class_name: 'Payment'
+end
```

Comment:
- [ ] The `name` attribute is required for creating for a user. Kindly add validation for its presence.

Reply:
Fixed